src/tests/ftest/pool/create_all_hw.py

```diff
@@ -81,21 +80,17 @@ def test_one_pool(self):
         self.log.info("Checking size of the pool")
         self.pool[0].get_info()
         tier_bytes = self.pool[0].info.pi_space.ps_space.s_total
-        self.assertLessEqual(
-                abs(self.scm_avail_bytes - tier_bytes[0]), self.delta_bytes,
-                "Invalid SCM size: want={}, got={}, delta={}".format(
-                    self.scm_avail_bytes, tier_bytes[0], self.delta_bytes))
-        self.assertLessEqual(
-                abs(self.smd_avail_bytes - tier_bytes[1]), self.delta_bytes,
-                "Invalid SMD size: want={}, got={}, delta={}".format(
-                    self.smd_avail_bytes, tier_bytes[1], self.delta_bytes))
+        self.assertLessEqual(abs(self.scm_avail_bytes - tier_bytes[0]), self.delta_bytes,
+                             "Invalid SCM size: want={}, got={}, delta={}".format(
+                             self.scm_avail_bytes, tier_bytes[0], self.delta_bytes))
+        self.assertLessEqual(abs(self.smd_avail_bytes - tier_bytes[1]), self.delta_bytes,
+                             "Invalid SMD size: want={}, got={}, delta={}".format(
+                             self.smd_avail_bytes, tier_bytes[1], self.delta_bytes))
```

Comment:
Was the previous format invalid? These look like they were okay

Reply:
It took me ages to get these to pass, yes they were incorrect.

Reply:
@phender  fixed this file in https://github.com/daos-stack/daos/pull/10718 so I've updated this PR and it no longer changes this file at all.